diff --git a/node_modules/tsup/dist/rollup.js b/node_modules/tsup/dist/rollup.js
index 65df1bf..f32a5d5 100644
--- a/node_modules/tsup/dist/rollup.js
+++ b/node_modules/tsup/dist/rollup.js
@@ -6717,7 +6717,7 @@ var parseCompilerOptions = (compilerOptions) => {
   const { options } = _typescript2.default.parseJsonConfigFileContent(
     { compilerOptions },
     _typescript2.default.sys,
-    "./"
+    _path2.default.resolve("./")
   );
   return options;
 };
